### DISCUSSION TITLE
Access 2003 plugin build?

### DISCUSSION THREAD

**Discussion by stickleprojects** [joyfullservice/msaccess-vcs-addin#289] [Ideas]
Hi
Any chance of an older build for this - I know the original BAS version is available but the optimisations and ease of use of this version would be amazing - currently running a bunch of 2003 databases with a 2 year-long project to migrate/update

**Comment by joyfullservice** [comment]
Great question! No, I don't have any version lower than Access 2010. Depending on the specifics of your project, there are a couple options you might consider:

- You could fork the repository and backport a version to Access 2003. The modular design of the add-in should make this a little easier, but there would be some functionality that would not be available in older versions of Access. I have never tried it, but my hunch is that probably 95% of it would work right out of the box in Access 2003.
- You could do an interim migration to initially move all the project databases up to Access 2010. Based on my experience, you might not have to change much, if anything, to get 2003 databases to run on Access 2010. (With Access 2010 you still have full ADP support, free runtime, etc...) If you are dealing with ADP project(s), Access 2010 is going to be the final version they will run on. This would allow you to use the latest releases of this add-in while you work through the migration/update process. Then you get the benefit of all your changes along the way being nicely saved in version control, and the ability to easily build from source as you test out various changes.

If your long-term goal is to migrate/update the databases to run on current versions of Access, I would personally lean towards option 2 where you can take full advantage of the VCS tools during the migration process.

Hope that helps!

**Reply by stickleprojects** [reply]
thanks for responding so quickly :) 
i'll review creating a fork on this
